pkgs/development/python-modules/pyuv/default.nix

```diff
@@ -2,13 +2,14 @@
 , buildPythonPackage
 , pythonAtLeast
 , fetchFromGitHub
+, fetchpatch
 , libuv
 }:
 
 buildPythonPackage rec {
   pname = "pyuv";
   version = "1.4.0";
-  disabled = pythonAtLeast "3.10"; # https://github.com/saghul/pyuv/issues/273
+  disabled = pythonAtLeast "3.11";
```

Comment:
Did you actually try to build it on Python 3.11?

Reply:
Yes. It fails without a patch.